resources/patches/openssl/openssl-3.0.13-create-dir-win.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/Configure b/Configure
+index 84cc409464..bdf4f3c460 100755
+--- a/Configure
++++ b/Configure
+@@ -3409,7 +3409,8 @@ sub absolutedir {
+     # We use realpath() on Unix, since no other will properly clean out
+     # a directory spec.
+     use Cwd qw/realpath/;
+-
++    
++    mkdir $dir unless -d $dir;
```

Comment:
This works around the issue, but it seems less than ideal to create a directory as a side effect of calling `absolutedir`. It looks like this issue has been fixed upstream already and backported to the 3.0.x branch https://github.com/openssl/openssl/commit/7b3eda56d7891aceef91867de64f24b20e3db212 I'm thinking we should pull in that patch instead.